### PR TITLE
add elastic cluster instructions

### PR DESCRIPTION
**Pull request by lennessyy**

### CODE REVIEW

File: modules/ha/pages/set-up-elastic-cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+$ gelasticcluster -p -m $<shared_storage_folder>
+----
+
+This operation empties the graph as it changes the data storage location.
+
+=== Load data into primary cluster
+Define schema and load data into the primary cluster.
+Perform your normal workloads on the primary cluster until you need to launch an elastic cluster to support computation-intensive workload.
+
+WARNING: Launching a elastic cluster requires that the primary cluster cannot be empty.
```
Comment (bowfengtigergraph) [inline modules/ha/pages/set-up-elastic-cluster.adoc:29]: a --> an

**Review by bowfengtigergraph — APPROVED** [review]
LGTM

File: modules/ha/pages/set-up-elastic-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+$ gelasticcluster -s -m <shared_storage_folder> -e <schema_folder>
+----
+
+=== Run workload on elastic cluster
+Now that the elastic cluster is provisioned, you can perform your workload on the cluster.
+
+You are only allowed to perform workloads that do not modify the data.
+For example, you can run OLAP queries that are read-only.
+
+If you attempt to perform an operation that would modify the data, the system blocks the operation to ensure that the primary cluster isn't affected by the elastic cluster in any way.
```
Comment (a-m-thomas) [inline modules/ha/pages/set-up-elastic-cluster.adoc:64]: Does the process break if it does this or does it just throw an error and let you keep working?

File: modules/release-notes/pages/index.adoc [hunk 3]
```diff
@@ -10,13 +10,13 @@ TigerGraph Server {page-component-version}.0 was released on Aug 31st, 2022.
...
 
 === Manageability
 
 
 === Security
-* Enhanced the xref:user-access:access-control-model.adoc#_role_based_access_control[role-based access control] system to allow data CRUD privileges to be granted at the type and attribute level.
+* Enhanced the xref:user-access:access-control-model.adoc#_role_based_access_control[role-based access control] system to allow data CRUD privileges to be granted at the type and attribute level (Preview).
 
 === Ecosystem integration
 * Added the ability to xref:data-loading:data-streaming-connector/azure-blob.adoc[stream data from Azure Blob Storage] to xref:data-loading:data-streaming-connector/index.adoc[Data Streaming Connector].
```
Comment (a-m-thomas) [inline modules/release-notes/pages/index.adoc:22]: to a Data Streaming Connector

**Review by a-m-thomas — APPROVED** [review]

File: modules/release-notes/pages/index.adoc (hunk 3 above)
Comment (lennessyy) [reply modules/release-notes/pages/index.adoc:22]: Here I am talking about Data Streaming Connector as a feature

**Pull request merged**